src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -341,7 +350,13 @@ export default {
 		},
 
 		statusMessage() {
-			return this.getStatusMessage(this.participant)
+			if (this.isInCall && this.participant.inCall && this.timeSpeaking) {
```

Comment:
Do we need both conditions here? 
I think `this.isInCall ` already checks whether the participant is on call or not and we don't have to check the value `inCall` specifically.

Reply:
First checks if current user is in call
Second - for the participant in this component
So we don't show timer, if user haven't said anything, or we are not in the call